src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getDetailsOfParents } from '../../helpers';
+
+export const PeoplePage: FC = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState('');
+  const { slug = '' } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(person => {
+        setPeoples(person);
+        setIsLoading(false);
+      })
+      .catch(() => {
+        setIsLoading(false);
+        setError('Something went wrong');
+      });
+  }, []);
+
+  const visiblePeoples = getDetailsOfParents(peoples);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      {isLoading
+        ? (
+          <Loader />
+        ) : (
+          <>
```

Comment:
There is unnecessary use of the fragment shorthand `<></>`. It could be removed, since it does not wrap multiple elements.